rails_app/db/schema.rb

```diff
@@ -29,17 +30,19 @@
     t.string "followed_currency_code"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
-    t.index ["follower_email", "followed_currency_code"], name: "unique_on_follower_+_followed_currency", unique: true
+    t.bigint "currency_id"
```

Comment:
Isn't this the same as the `id` column of this table that is automatically created when generating the `currencies` table?

Reply:
This is to ensure db-side that there aren't duplicate records, e.g. multiple pairs (X, Y) of user X and currency Y. Also, it's useful when loading any currencies page where we need to check if a user follows a currency.

Reply:
This is in the `currency_followings` table, it's the foreign key for the `currencies` table. Since github collapses lines, initially I did not notice that and had to go check the actual schema file. 
Did you not notice that too or am I missing something?

Reply:
Oh, my bad. 😄 It was that, yes.